packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-submission.jsx

```diff
@@ -170,9 +170,11 @@ const POISubmission = observer(
                                 handleSubmit={handleViewComplete}
                                 latest_status={identity_last_attempt}
                                 selected_country={selected_country}
+                                handleSelectionNext={handleSelectionNext}
                             />
                         ) : (
                             <IdvDocumentSubmit
+                                handleSelectionNext={handleSelectionNext}
```

Comment:
now that we're using POIContext we don't need to pass this function to IdvFailed or IdvDocumentSubmit
You can access the context directly